Dockerfile

```diff
@@ -4,57 +4,64 @@
 # https://github.com/GameServerManagers/LinuxGSM-Docker
 #
 
-FROM ubuntu:16.04
-LABEL maintainer="LinuxGSM <me@Danielgibbs.co.uk>"
+FROM ubuntu:18.04
+-LABEL maintainer="LinuxGSM <me@Danielgibbs.co.uk>"
```

Comment:
Hyphen before label instruction?

Reply:
Sorry, my bad wasn't supposed to be there, thanks for noticing.
I updated it already.